syntax_checkers/go/govet.vim

```diff
@@ -23,7 +23,7 @@ function! SyntaxCheckers_go_govet_IsAvailable() dict
 endfunction
 
 function! SyntaxCheckers_go_govet_GetLocList() dict
-    let makeprg = self.getExecEscaped() . ' vet'
+    let makeprg = self.makeprgBuild({ 'args_after': 'vet' })
```

Comment:
I looked at `g:SyntasticChecker._getOpt` briefly, seems it's safe to just use `args`. Using `args`, unless you tell me otherwise.